_posts/2024-05-28-pyos-newsletter-june-2024.md

```diff
@@ -0,0 +1,112 @@
+header:
+  overlay_color: "#666"
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+## <i class="fa-regular fa-comments"></i> PyCon was incredible!
+We feel so incredibly fortunate to have connected with so many of you at PyCon! While we're still recovering from a week of summits, talks, and sprints, we wanted to share this sentiment from our Executive Director and Founder, [Leah Wasser](https://github.com/lwasser), which she shared on [her Fosstodon account](https://fosstodon.org/@leahawasser):
+<figure>
+    <a href="/images/blog/2024/may/leah-pycon.png">
+    <img src="/images/blog/2024/may/leah-pycon.png" style="max-width:100%" alt="Screenshot of a Fosstodon post from Leah Wasser, reading: In my first few moments at pycon I ran into some of my conda friends & was quickly reminded of why this conference is so powerful.  Python is more than a programming language. It's a community of people who I can geek out with about the most obscure topics. I belong. People care. They care a lot about their work. They care about helping each other learn.  While this is a tech conference, it's really people that make python the vibrant communty that it is.">
+    </a>
+</figure>
+
+## <i class="fa-solid fa-handshake-simple"></i> See you at SciPy
+[SciPy 2024](https://www.scipy2024.scipy.org/) is just around the corner, and we can't wait to see you there! We're pulling together our pyOpenSci Guide to SciPy, similar to the guide we did for [PyCon 2024](https://www.pyopensci.org/blog/pyos-guide-to-pyconus-2024.html), and wanted to give you a preview of some of the tutorials and talks being given by pyOpenSci Community members!
+
+### Tutorials
+#### [3D Visualization with PyVista](https://cfp.scipy.org/2024/talk/GKGRWE/)
+Led by [Tetsuo Koyama](https://cfp.scipy.org/2024/speaker/WY7NA9/), [Alexander Kaszynski](https://cfp.scipy.org/2024/speaker/3RXHG8/), [Bill Little](https://cfp.scipy.org/2024/speaker/FDVWDN/), and [Bane Sullivan](https://cfp.scipy.org/2024/speaker/NEC33M/), this tutorial demonstrates [PyVista's](https://github.com/pyvista/pyvista) latest capabilities and bring a wide range of users to the forefront of 3D visualization in Python, including:
+
+* Use PyVista to create 3D visualizations from a variety of datasets in common formats.
+* Overview the classes and data structures of PyVista with real-world examples.
+* Be familiar of the various filters and features of PyVista.
+* Know which Python libraries are used and can be used by PyVista (meshio, trimesh etc).
+* We see this tutorial catering to anyone who wants to visualize data in any domain, and this ranges from basic Python users to advanced power users.
+
+#### [Create Your First Python Package: Make Your Python Code Easier to Share and Use](https://cfp.scipy.org/2024/talk/QT9GBY/)
+Led by pyOpenSci's Executive Director and Founder, [Leah Wasser](https://github.com/lwasser), in this tutorial session you'll learn how to:
```

Comment:
i do intend to add others to the tutorial. i just need help. maybe i'll post now but if you can help find helpers too i'd appreciate it!

Reply:
@lwasser that makes sense - do you want me to add a call for volunteers in the newsletter?

Reply:
yes that would be wonderful Jesse. thank you. i am not sure how many will attend but you saw how the online workshop went. i like having 4 people around if possible including myself. You can even mention the time and room and such and how to reach out if they can help!